meson.build

```diff
@@ -1,4 +1,7 @@
-project('ksmbsd-tools', 'c', version: '3.3.2', default_options: 'c_std=gnu99')
+project('ksmbsd-tools', 'c',
+  version: run_command(find_program('awk'), '''/define KSMBD_TOOLS_VERSION / \
+    { gsub(/"/,"",$3); printf "%s", $3 }''', 'include/version.h', check: true).stdout(),
```

Comment:
Since ksmbd-tools is Linux only, should be fine. However, this does rely on a host awk. I would make sure this builds with BSD AWK and not just GNU AWK (OpenWrt is builtable on macOS).

Reply:
@neheb, this works on the One True Awk. It's up to you to test any other particular implementations that differ.

Reply:
Just making sure. I've seen such issues before. I don't speak awk.